### PR TITLE
Update docs on how to add new releases

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
`generator add` should be used to add releases for all not predefined plugins. This PR updated the documentation to make this clear.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-marketplace/issues/74

### CODE REVIEW

**Comment by hanzei** [comment]
Good question @larkox. Given that sometime `make plugins.json` should be used and sometime `generator add` I'm not quite sure what mattermod should output. Maybe link this documentation?

**Comment by larkox** [comment]
It would break the flow for those that just want to copy paste those lines, but yes, link to this documentation may be needed. I wonder whether a small explanation on the message (for those that have done it over and over but fail to remember the syntax) on the two options would be nice too.

File: README.md [hunk 1]
```diff
@@ -52,20 +52,51 @@ export BUILD_UPSTREAM_URL=https://api.integrations.mattermost.com
...
+```
+`generator add` supports additional flags. See `generator add --help` for more details.
+
+Make sure to double check the `diff` of `plugins.json` to ensure the release get added correctly.
+
+After you are satisfied with the changes, run
+```
+make generate
+```
+to update `data/statik/statik.go` and commit both the changes in `plugin.json` and `data/statik/statik.go`.
```
[the remark this answers is not in the document]
Comment (hanzei) [reply README.md:73]: I assume that would render to the same output, right?

**Comment by hanzei** [comment]
@jfrerich @larkox After thinking a bit more about this PR, I don't consider supporting two ways of adding a release valuable any longer. Hence, I'm leaning toward deprecating `make plugins.json` in favour of using `generate add`. I've updated the doc to reflect that proposal. Let me know what you think.

**Review by larkox — APPROVED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jfrerich) [reply README.md:73]: The suggestion was to combine the two separate sentences and remove the `make generate` from in between them.  This is a nit that is 0/5 for readibilty.

[the remark this answers is not in the document]
Comment (hanzei) [reply README.md:73]: Makes sense :+1:

**Pull request merged**